staging/src/k8s.io/sample-controller/pkg/apis/example/register.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package example
+
+const (
+	GroupName = "example.controller.code-generator.k8s.io"
```

Comment:
how about only `example.controller.k8s.io`?

Reply:
I chose this as the sampe-apiserver changes that @sttts has made use `example.apiserver.code-generator.k8s.io`. I'm happy to change this to whatever makes sense though :)